indra/newview/pipeline.cpp

```diff
@@ -1068,6 +1075,9 @@ void LLPipeline::refreshCachedSettings()
         LLViewerShaderMgr::instance()->clearShaderCache();
         LLViewerShaderMgr::instance()->setShaders();
     }
+    RenderHeroProbeUpdateRate = gSavedSettings.getS32("RenderHeroProbeUpdateRate");
+    RenderHeroProbeConservativeUpdateMultiplier = gSavedSettings.getS32("RenderHeroProbeConservativeUpdateMultiplier");
+
```

Comment:
Prefer to not use connectRefreshCachedSettings and statics, but use static LLCachedControl.